packages/components/src/components/list/List.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module '@enykeev/react-virtualized/dist/es/List' {
+    import { ListProps } from '@enykeev/react-virtualized/dist/es/List';
+
+    export default class List extends React.PureComponent<ListProps> {}
+}
```

Comment:
your current `.d.ts` files are just disappearing the ts errors but if you hover on `List` component in `data-table.tsx,` you can see that the type is `any`! after applying these changes you will be able to see the type for List ( I wasn't sure about `ref` and `deferredMeasurementCache` so I assigned `any` to them for now: 


![image](https://user-images.githubusercontent.com/100833613/223760808-09ad42e5-24cb-46c3-87bf-f6104170d75e.png)

the same thing goes for CellMeasurerCache and CellMeasurerCache. you need to type them as well. please let me know if you need help with typing them as well :pray: 

```suggestion

declare module '@enykeev/react-virtualized/dist/es/List' {
    import { ListProps } from '@enykeev/react-virtualized/dist/es/List';
    import type { Grid } from 'react-virtualized';

    type TRowRenderer = {
        style:CSSProperties;
        index: number;
        key:string;
        parent:React.RefObject<Grid>;
    }

    type TList = {
        autoHeight: boolean,
        className?: string,
        deferredMeasurementCache: any;
        height: number;
        overscanRowCount: number;
        ref: any;
        rowCount: number;
        rowHeight: number;
        rowRenderer: (props: TRowRenderer) => React.ReactNode;
        scrollingResetTimeInterval: number;
        scrollTop: number;
        width: number;
    }

    export const List =  ({
        autoHeight,
        className,
    }: TList): JSX.Element => JSX.Element;
    export default List;
}

}
```